config/manager/manager.yaml

```diff
@@ -15,6 +15,8 @@ spec:
       labels:
         control-plane: controller-manager
     spec:
+      serviceAccountName: manager
+      automountServiceAccountToken: true
```

Comment:
Q: is this set to false and best practice, when the default SA is used?

Reply:
If I understand the question correctly, yes, it's best-practice to set this to false when not using a custom serviceAccount to prevent the accidental usage of the default serviceAccount

In general it's best-practice to not use the default serviceAccount anyways